contracts/AllDay.cdc

```diff
@@ -720,6 +994,42 @@ access(all) contract AllDay: NonFungibleToken {
             return "https://nflallday.com/moments/".concat(self.id.toString())
         }
 
+        access(all) fun getBadges(): [Badge]?{
```

Comment:
just curious, do we maybe want badges in traits metadata view, or was that left out intentionally for now?

Reply:
good point, we should add it there

Reply:
added [here](https://github.com/dapperlabs/nfl-smart-contracts/pull/32/commits/9bcbf2a0524075dec59038eff37dc74a9a277c1c)